service/src/main/java/bio/terra/workspace/service/resource/controlled/cloud/azure/database/CreateAzureDatabaseStep.java

```diff
@@ -154,9 +147,10 @@ private Optional<String> waitForCreateDatabaseContainer(CoreV1Api aksApi, String
   }
 
   private void logPodLogs(CoreV1Api aksApi, String podName, Integer tailLines) {
+    var logContext = Map.of("workspaceId", workspaceId, "podName", podName);
     try {
       var pod = aksApi.readNamespacedPod(podName, aksNamespace, null);
-      logger.info("Pod {} final status: {}", podName, pod.getStatus());
+      logger.info("Pod final status: {}", pod.getStatus(), logContext);
```

Comment:
The change in logging format makes this much easier to read and query. The message is not prefixed by a huge string which causes the important parts to be clipped in the logs viewer. The workspace id and pod name are included in the json payload for better querying.